### DISCUSSION TITLE
Building itowns-starter

### DISCUSSION THREAD

**Discussion by dayjournal** [iTowns/itowns#1929] [Ideas]
Hi everyone!
I have built a build environment that allows me to easily start "iTowns" with a combination of "iTowns" and "Vite."
Please check it out!

https://github.com/dayjournal/itowns-starter

![スクリーンショット 2022-09-11 23 39 05](https://user-images.githubusercontent.com/7871642/189533566-89457d24-13de-45d4-ba70-14ba27962226.png)

**Comment by mgermerie** [comment]
Thank you for this project, we will showcase it on a list of projects using iTowns :+1:

**Reply by dayjournal** [reply]
Hi, @mgermerie Thank you!
